src/zNAClient.ts

```diff
@@ -0,0 +1,37 @@
+import { ethers } from 'ethers';
+
+import zDAOCore from './config/constants/abi/zDAOCore.json';
+import { zDAOId, zNA, zNAConfig } from './types';
+import { t } from './utilities/messages';
+
+class zNAClient {
```

Comment:
This file should be in it's own folder `zDAORegistry` and the class renamed to `zDAORegistryClient`.

This is the client which will interact with the smart contracts.